### PR TITLE
fix: At Success modal is a preview of iterated art (good), and on the listing is the default. (bad)

### LINKED ISSUE #9804: At Success modal is a preview of iterated art (good), and on the listing is the default. (bad)
On the Success modal is a preview of iterated art (good), and on the listing is the default. (bad)

https://github.com/kodadot/nft-gallery/assets/5887929/5103e9cf-da4d-49be-9d2d-fb54979715bc

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/9800#issuecomment-2003669727_

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

fix until https://github.com/kodadot/private-workers/issues/87 is implemented and  "setMetadata" is done earlier  as @preschian  mentioned

> But, I think it can solve the problem if we also put metadata in our minting step. Since it works on the "success modal". In https://github.com/kodadot/private-workers/issues/87#issuecomment-1997444473, I put "setMetadata" in the steps 2 and 6

- [x] Closes #9804

## Needs QA check

- @kodadot/qa-guild please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

https://github.com/kodadot/nft-gallery/assets/44554284/1b84cd9e-203e-42af-acd5-72b796794fa8

### CODE REVIEW

**Comment by yangwao** [comment]
cc @prury ?

File: components/common/listingCart/shared/ListingCartItemDetails.vue [hunk 1]
```diff
@@ -3,11 +3,22 @@
...
         <div>
+          <BaseMediaItem
+            v-if="nft.animationUrl"
+            class="border border-k-shade image is-48x48"
+            :class="{ 'opacity-50': discarded }"
+            :alt="nft?.name"
+            :animation-src="sanitizeIpfsUrl(nft.animationUrl)"
+            mime-type="text/html"
+            preview
+            is-detail />
```
Comment (Jarsen136) [inline components/common/listingCart/shared/ListingCartItemDetails.vue:14]: What if the mime-type of nft is vedio or 3d-object ?

Comment (hassnian) [reply components/common/listingCart/shared/ListingCartItemDetails.vue]: fixed [f5abf6c](https://github.com/kodadot/nft-gallery/pull/9833/commits/f5abf6c1438c22590001dfb058d244c7a39dab46)

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 9.43 USD/DOT ~ 5.302 $DOT
🧗 [16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD](https://polkadot.subscan.io/account/16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)
🔗 [0xb441bb54b0e3a450e2b0e45ce71631b21e250bae853cef542fa8b243ab971ef2](https://polkadot.subscan.io/extrinsic/0xb441bb54b0e3a450e2b0e45ce71631b21e250bae853cef542fa8b243ab971ef2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**